.github/workflows/cd-executor.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Release
+
+on:
+    workflow_call:
+      inputs:
+        target-branch:
```

Comment:
We don't need the branch as an input. When triggering the workflow itself, there's an option to select the branch.

Reply:
fixed in [d4ede32](https://github.com/wso2/open-healthcare-prebuilt-services/pull/45/commits/d4ede32dcac99e3d755eeaddbfc9987a4c049cb2)